composer.json

```diff
@@ -10,10 +10,9 @@
     ],
     "require": {
         "php": "^7.3 || ^8.0.0",
-        "webmozart/path-util": "^2.2"
+        "symfony/filesystem": "^5.4"
```

Comment:
I'm a bit vary on introducing `symfony/*` dependencies, since they are inherently unsafe from a dependency PoV: 

https://github.com/symfony/filesystem/blob/731f917dc31edcffec2c6a777f3698c33bea8f01/composer.json#L19

See also https://github.com/webmozarts/glob/issues/57

Reply:
@Ocramius What's the stance of the Symfony team on that?

Reply:
No idea: tbh, including their classes (with a notice here) is probably the safest bet.

Reply:
Right now merging and releasing this is still better than keeping the abandoned repo in there as dependency.
Along the lines quite a few libraries now generate the warning report here:

> Package webmozart/path-util is abandoned, you should avoid using it. Use symfony/filesystem instead.

Reply:
But in that case a higher support for also Symf 4/5/6 might be good.

Or indeed the classes are just copied - as long as you keep the original copyright (license header), it will be fine.
Thats even easier and will have less dependency issues with symfony and changing their signature/code.

Reply:
> Right now merging and releasing this is still better than keeping the abandoned repo in there as dependency.

Not really a problem until a security issue occurs.

I suggest sending a patch with the necessary upstream dependency code, license headers and tests included.